### ISSUE TITLE
Autocompletion is slow with all 0.25 versions on Windows

### ISSUE THREAD

**Issue by Daleck7** [hashicorp/terraform-ls#775]
### Server Version

```
0.25.2
platform: windows/amd64
go: go1.17.5
compiler: gc
```

### Terraform Version

```
Terraform v1.1.4
on windows_amd64
```

### Client Version

```
Visual Studio Code 1.63.2
```

### Expected Behavior

When writing terrafrom code, autcompletion si normally instant and hitting CTRL+Space drop a menu with some choices instantly.

### Actual Behavior

Autocompletion not working on first try or show "Loading..." box. Same thing when hitting CTRL+Space.
When I rollback to version 0.24 of the terraform-ls, everything shows up instantly. If I go back to 0.25 (any sub release), it's slow again. I can reproduce the same behavior on 3 different PC. (Works fine on MacOS and Linux).

### Steps to Reproduce
Please list the full steps required to reproduce the issue, for example:
1. Open a folder in VSCode
2. Create the main.tf file
3. Start writing and wait for autocompletion.
4. Install terraform-ls 0.24, do same thing and evertyhing is good.

**Comment by radeksimko** [comment]
Hi @Daleck7 
This has likely the same root cause as https://github.com/hashicorp/terraform-ls/issues/768 and we're already working on a fix of that bug as part of https://github.com/hashicorp/terraform-ls/issues/719

However - for some added peace of mind - would you mind sharing the log from `Output -> HashiCorp Terraform`?

Thanks!

**Comment by Daleck7** [comment]
Sure, here is the output when starting with the terraform-ls 0.25.2 included in the wrapper. For logging purpose I've juste start writing a variable and a resource.
[output.txt](https://github.com/hashicorp/terraform-ls/files/7950972/output.txt)

**Comment by radeksimko** [comment]
The relevant PR #782 which should address the underlying problem described here was merged and will be part of an upcoming 0.26.0 release. We plan to release 0.26.0-beta1 some time next week (which includes the mentioned patch), which will also become available as part of our [VSCode preview extension](https://marketplace.visualstudio.com/items?itemName=HashiCorp.terraform-preview).

Thank you for your patience.
